docs/modules/ROOT/pages/explanations/decisions/mariadb.adoc

```diff
@@ -95,12 +97,15 @@ Disadvantages::
 * Maintained by https://repo-tracker.com/r/gh/mariadb-operator/mariadb-operator[a single maintainer] only, if he leaves the project, we would have to fork and maintain it ourselves
 * Restores database to new instance, like Stackgres does in PostgreSQL, which is not ideal for us, but we can work around it
 * We would need to write and maintain a new Project Syn Commodore Component to deploy this operator
+* Potentially more difficult to move the SPKS project to AppCat
 
 === Analysis - Bitnami Helm Chart
 
 https://github.com/bitnami/charts/tree/master/bitnami/mariadb-galera[Bitnami Helmchart^].
 
-Bitnami Helm Charts are solid, actively maintained, and we already have experience integrating it into our offerings. 
+Bitnami Helm Charts are solid, actively maintained, and we already have experience integrating it into our offerings.
+It is not as feature rich as the operator though it has much lower risk of orphanage.
+Backup and automatic restore will require more time to engineer unlike the operator.
```

Comment:
I feel like this directly contradicts the previous line?

So the operator does have more features than the helm chart? Specifically backups and restore?